cost/turbonomics/buy_reserved_instances_recommendations/aws/README.md

```diff
@@ -0,0 +1,28 @@
+# Turbonomics Buy Reserved Instances Recommendations AWS
+
+## What it does
+
+The Turbonomics Buy Reserved Instances Recommendations AWS policy utilizes Turbonomics [Credentials]https://turbonomic.com/api/v3/markets/{market_uuid}/actions endpoint to provide AWS purchasable RI recommendations for Reserved Instance coverage. From these recommendations we provide monthly savings estimates based on Turbonomics per hour costs
```

Comment:
"The Turbonomics Buy Reserved Instances Recommendations AWS policy utilizes Turbonomic Actions API endpoint to provide AWS RI purchase recommendations.